test/pamela/HTN/symbol-ref-htn.main.htn.edn

```diff
@@ -0,0 +1,372 @@
+    :type :mode-ref}},
+  :display-args [4 5 (mode-of locations :sea)],
+  :display-name "Method 3",
+  :edges [],
+  :incidence-set #{:hedge-81},
+  :name "method3",
+  :type :htn-expanded-nonprimitive-task,
+  :uid :henpt-51},
+ :hpt-38
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :state-variable, :name state-var}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :state-variable, :name state-var}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= state-var method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-65],
+  :incidence-set #{},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-38},
+ :hpt-39
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :field-ref, :names [field3]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :field-ref, :names [field3]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= field3 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-66],
+  :incidence-set #{:hedge-65},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-39},
+ :hpt-40
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :field-ref, :names [field2 field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :field-ref, :names [field2 field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= field2.field1-1 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-67],
+  :incidence-set #{:hedge-66},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-40},
+ :hpt-41
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :method-arg-ref, :names [pclass-instance-arg field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :method-arg-ref, :names [pclass-instance-arg field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= pclass-instance-arg.field1-1 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-68],
+  :incidence-set #{:hedge-67},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-41},
+ :hpt-42
+ {:args [42],
+  :argsmap {"arg" 42},
+  :display-args [42],
+  :display-name "Simple Method",
+  :edges [:hedge-69],
+  :incidence-set #{:hedge-68},
+  :name "simple-method",
+  :type :htn-primitive-task,
+  :uid :hpt-42},
+ :hpt-43
+ {:args [{:type :state-variable, :name state-var}],
+  :argsmap {"arg" {:name state-var, :type :state-variable}},
+  :display-args [state-var],
+  :display-name "Simple Method",
+  :edges [:hedge-70],
+  :incidence-set #{:hedge-69},
+  :name "simple-method",
+  :type :htn-primitive-task,
+  :uid :hpt-43},
+ :hpt-44
+ {:args [100],
+  :argsmap {"arg" 100},
```

Comment:
Here, we are compiling inline the field value of `field3` into the `:args` (and `:display-args`).  
So the field inlining is more pervasive than just in `:display-args`